spec/tools/fix_auth/auth_model_spec.rb

```diff
@@ -6,19 +6,13 @@
 
 RSpec.describe FixAuth::AuthModel do
   let(:pass)    { "password" }
-  let(:enc_old) { ManageIQ::Password.new.encrypt(pass, "v2", legacy_key) }
+
+  let(:enc_old) { ManageIQ::Password.encrypt(pass, legacy_key) }
   let(:enc_new) { ManageIQ::Password.encrypt(pass) }
   let(:enc_bad) { "v2:{5555555555555555555555==}" }
 
   let(:legacy_key) { ManageIQ::Password::Key.new }
-
-  before do
-    ManageIQ::Password.keys["alt"] = legacy_key
-  end
-
-  after do
-    ManageIQ::Password.clear_keys
-  end
+  let(:options)    { {:legacy_key => legacy_key } }
```

Comment:
ditto on either passing in the full text or changing this `options` name

Reply:
aah - after reading through more tests I understand why you did `options`, but manually passing in both seems pretty clear. up to you